Source/SPUD/Public/SpudSubsystem.h

```diff
@@ -222,6 +222,9 @@ class SPUD_API USpudSubsystem : public UGameInstanceSubsystem, public FTickableG
 	virtual void Initialize(FSubsystemCollectionBase& Collection) override;
 	virtual void Deinitialize() override;
 
+	void HandleLevelUnloaded(ULevel* Level);
+	void HandleLevelLoaded(FName LevelName);
+		
```

Comment:
Since these new functions are public, they need to be a little clearer about what they do and why someone might want to call them. They also need documentation comments (because they're not self-explanatory), and maybe even exposing to Blueprints like the rest of the API?

The best pattern might be to make `HandleLevel[Un]loaded` protected instead, and then expose different passthrough public functions which are more explicitly named & BP-available such as `NotifyLevelLoadedExternally` or something like that. I'm a big fan of method names being as self-evident as possible so that when you're scrolling through auto-complete you don't have to reach for a manual unless you need some finer details.

Reply:
I'm not attached to the names of these functions (in fact, I had them named more along the lines of your suggestion but at the last moment decided to change them to be more of a symmetrical pair - but happy to change them back as I agree with your rationale), so rather than add passthrough functions, I will just rename these functions as you suggested. Will also expose to BP and add docs.